salsa_dancing_molecules/simulations/argon.py

```diff
@@ -1,6 +1,7 @@
 """Helpers to run an example simulation on Argon."""
```

Comment:
Do the edits in this file effect the bulk calculations? It looks like edits that were commited by mistake, but I do not mind them since we do not use this file anymore.